gen/generate.go

```diff
@@ -105,6 +105,10 @@ func Generate(m *compile.Module, o *Options) error {
 	genBuilder := newGenerateServiceBuilder(importer)
 
 	generate := func(m *compile.Module) error {
+		if err := isDuplicateFileAfterNormalization(m.ThriftPath); err != nil {
```

Comment:
This name is a bit of a mouthful, consider renaming to something more succinct.

Reply:
okay